src/main/java/org/primeframework/mvc/workflow/StaticResourceWorkflow.java

```diff
@@ -108,7 +115,7 @@ public void perform(WorkflowChain workflowChain) throws IOException {
    * @param request  The request
    * @param response The response
    * @return True if the resource was found in the classpath and if it was successfully written back to the output stream. Otherwise, this returns
```

Comment:
```suggestion
   * @return True if the resource was found in the static resources directory or classpath, and it was successfully written back to the output stream. Otherwise, this returns false if the resource does not exist or when retrieval is not allowed by {@link StaticResourceFilter} or {@link StaticClasspathResourceFilter} configurations.
```